decidim-proposals/app/controllers/decidim/proposals/proposal_votes_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # Exposes the proposal vote resource so users can vote proposals.
+    class ProposalVotesController < Decidim::Proposals::ApplicationController
+      before_action :authenticate_user!
+
+      def create
+        @proposal = Proposal.find(params[:proposal_id])
+
+        set_votes_count_classes
```

Comment:
I'm reusing the same code for both proposals index and show view. The code is slightly different because they use different css classes (i.e. in proposals#show the buttons are bigger). I didn't find a better solution for this. Any ideas?